### ISSUE TITLE
Why `owner_name`?

### ISSUE THREAD

**Issue by janpio** [Microsoft/fastlane-plugin-appcenter#12]
It is unexpected that `owner_name` is needed additionally to `api_token` (in comparison to the `hockey` action for HockeyApp for example). 

What is the reason behind this requirement?
Only for the [API path](https://github.com/Microsoft/fastlane-plugin-appcenter/blob/master/lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb#L44)?

**Comment by lumaxis** [comment]
@janpio I know it doesn't seem intuitive but the `app_name` alone doesn't identify an app globally, the name is only unique per user. And since you in theory could have access to two or more apps with the same `app_name`, owned by different accounts, we use the `owner_name/app_name` combination to uniquely identify an app across App Center.

**Comment by janpio** [comment]
Oh, so my account (or better API token) can also have access to a similarly named app from another "owner_name"? Interesting.

**Comment by lumaxis** [comment]
Yes, exactly. If I invite you to my app with the same name and give you "Manager" or "Developer" access, you would also be able to upload new releases to it. Therefore, it's important we uniquely identify the app.
